### PR TITLE
Update New Brizy UI Style

### PR DESCRIPTION
**Pull request by JekaPolihovic**
### Changelog

* Updated: Update New Brizy UI Style

----
### Components Updates to style Brizy 

- [x] Form (Label, padding )
- [x] Input
- [x] Textarea
- [x] Select
- [x] Upload #9
- [x] Button (default, primary)
- [x] Tag
- [x] Typography (H2, Paragraph, Title, Text)
- [x] Badge
- [x] Tabs [https://prnt.sc/rqu2xu](https://prnt.sc/rqu2xu)
- [x] Button (**Link**) 
- [x] Menu [https://projects.invisionapp.com/d/main#/console/17132809/400088324/preview](https://projects.invisionapp.com/d/main#/console/17132809/400088324/preview)
- [x] Gray Button [https://projects.invisionapp.com/d/main#/console/17132809/400088323/preview](https://projects.invisionapp.com/d/main#/console/17132809/400088323/preview)
- [x] Checkbox [https://projects.invisionapp.com/d/main#/console/17132809/400417320/preview](https://projects.invisionapp.com/d/main#/console/17132809/400417320/preview)
- [x] Input **Search** [https://projects.invisionapp.com/d/main#/console/17132809/400416250/preview](https://projects.invisionapp.com/d/main#/console/17132809/400416250/preview)
- [x] _**Dark SideBar**_
- [x] Devider [https://prnt.sc/rqu8sa](https://prnt.sc/rqu8sa)
- [x] Tooltip [https://prnt.sc/ruvbub](https://prnt.sc/ruvbub)
- [x] Table [https://projects.invisionapp.com/d/main#/console/17132809/400414454/preview](https://projects.invisionapp.com/d/main#/console/17132809/400414454/preview)
- [ ] Icons
- [x] brizy-publish-dropdown [https://projects.invisionapp.com/d/main#/console/17132809/401184672/preview](https://projects.invisionapp.com/d/main#/console/17132809/401184672/preview)
- [x] Multiple Upload gallery [https://prnt.sc/rw2tba](https://prnt.sc/rw2tba)
- [x] Card image  [https://prnt.sc/rw6w9i](https://prnt.sc/rw6w9i)

### CODE REVIEW

File: less/index.less [hunk 1]
```diff
@@ -1,15 +1,258 @@
...
+@import "brizy-textarea.less";
+@import "brizy-upload.less";
+@import "brizy-select.less";
+@import "brizy-button.less";
+@import "brizy-tag.less";
+@import "brizy-divider.less";
+@import "brizy-menu.less";
+@import "brizy-tabs.less";
 
-@primary-color: #1890ff; // primary color for all components
```
Comment (GheorgheP) [inline less/index.less:3]: You should define varaibales in a separate file then import it here after
```less
@import "~antd/dist/antd.less";
```

Then you need to import elements specific styling, so they would have access to the updated variables.

**Comment by GheorgheP** [comment]
- [ ] Upload ui problem:

**Comment by GheorgheP** [comment]
- [ ] Tags spacing

**Comment by GheorgheP** [comment]
- [ ] Does text and textarea have outline on focus?

File: tsconfig.json [hunk 2]
```diff
@@ -29,4 +29,4 @@
         "**/*.test.ts",
         "**/*.test.tsx"
     ]
-}
\ No newline at end of file
+}
```
Comment (GheorgheP) [inline tsconfig.json:32]: @JekaPolihovic, I think you can remove this part.

File: tsconfig.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
         "moduleResolution": "node",
         "esModuleInterop": true,
         "experimentalDecorators": true,
-        "jsx": "react",
+        "jsx": "preserve",
```
Comment (oleg-brizy) [inline tsconfig.json:12]: Why have you changed it back to `preserve`?
https://github.com/bagrinsergiu/brizy-ui/pull/10/files
cc: @GheorgheP

**Pull request merged**